src/Bicep.Decompiler/Exceptions/ConversionFailedException.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Newtonsoft.Json;
+using System;
+
+namespace Bicep.Decompiler.Exceptions
+{
+    public class ConversionFailedException : Exception
+    {
+        public ConversionFailedException(string message, IJsonLineInfo jsonLineInfo)
+            : base(FormatMessage(message, jsonLineInfo))
+        {
+        }
+
+        private static string FormatMessage(string message, IJsonLineInfo jsonLineInfo)
+            => $"[{jsonLineInfo.LineNumber}:{jsonLineInfo.LinePosition}]: {message}";
```

Comment:
>$"[{jsonLineI [](start = 15, length = 13)

This makes me wonder if we're going to log errors in the same format as the compiler (there's information loss because `jsonLineInfo` is thrown away here).